Formula/fd.rb

```diff
@@ -0,0 +1,18 @@
+class Fd < Formula
+  desc "Simple, fast and user-friendly alternative to find."
+  homepage "https://github.com/sharkdp/fd"
+  url "https://github.com/sharkdp/fd/archive/v1.0.0.tar.gz"
+  sha256 "711ecfd34640fb4d840567231992102aa55da471aaafbffb7c2b76073872092e"
+  head "https://github.com/sharkdp/fd.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/fd"
+  end
+
+  test do
+    system "#{bin}/fd"
```

Comment:
@ilovezfs from the formula template itself: `it's enough to just replace "false" with the main program this formula installs, but it'd be nice if you were more thorough.`. It means it's mandatory to actually verify by functionality of the software, but it would be nice to.